addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -4,6 +4,24 @@ import contentMenu from 'website.contentMenu';
 import weWidgets from 'wysiwyg.widgets';
 import {_t} from 'web.core';
 
+weWidgets.LinkPopoverWidget.include({
+    /**
+     * @override
+     */
+    start: function () {
+        // hide popover while typing on mega menu
```

Comment:
I'd like an uppercase letter to start the comment but... can't force you to accept my convention :D

Reply:
With an uppercase comes a period at the end, I always hated those comments except for docstring. You can't force me except with :moneybag: 
More seriously, I can change it if you want to enforce/follow a common guideline for inline comments